docker/dbt/jaffle_shop/dbt_project.yml

```diff
@@ -22,6 +22,9 @@ models:
     materialized: table
     staging:
       materialized: view
+    table_drop:
+      materialized: table
+      +on_table_exists: dupa
```

Comment:
I assume this is an unintended change.

Reply:
thanks, that's a not polite word usually used to test things in Polish :D